packages/remix/src/ssr/getAuth.ts

```diff
@@ -5,7 +5,7 @@ import { authenticateRequest } from './authenticateRequest';
 import type { GetAuthReturn, LoaderFunctionArgs, RootAuthLoaderOptions } from './types';
 import { interstitialJsonResponse, unknownResponse } from './utils';
 
-type GetAuthOptions = Pick<RootAuthLoaderOptions, 'apiKey' | 'secretKey'>;
+type GetAuthOptions = Pick<RootAuthLoaderOptions, 'apiKey' | 'secretKey' | 'loadUser'>;
```

Comment:
Clerk also supports `loadOrganization` & `loadSession` flags. So my suggestion is passing them all to getAuth.